activerecord/lib/active_record/relation/calculations.rb

```diff
@@ -133,6 +133,7 @@ def calculate(operation, column_name)
     #   Person.pluck(:id)
     #   # SELECT people.id FROM people
     #   # => [1, 2, 3]
+    #   # Use Person.ids if only ids needed.
```

Comment:
I'd say we should replace the example all together. Doesn't make much sense to showcase the method in a way you wouldn't want to use it.